### ISSUE TITLE
Holdouts with limits data points

### ISSUE THREAD

**Issue by samjameswaller** [InseadDataAnalytics/INSEADAnalytics#93]
Hi all - for Yahoo/Tumblr - if we only have ~40 data points, does it make sense to have a holdout / testing sample? My concern is if we hold out any data, we won't have enough to train the model properly. Thanks! Sam

**Comment by VarunKShetty** [comment]
Hi @samjameswaller 

Two points here: 

1. Since this is a time-series model, we will hold out only the latest observations. Say the last (latest) 5 observations or the last 2 observations and NOT a randomly chosen set of observations. This is because, of the inherent structure of time-series data -- we would assume that we know the past and want to predict the future. 
2. Since we have only 40 data points, we will have to hold out a much smaller sample. However, we would miss out on being able to take advantage of the law of large numbers when dealing with residuals in this case. For example, consider the case that you hold out only 3 data points and one of them is an outlier -- you would end up with a large holdout error even if the model itself was a pretty good fit. 

tl;dr: There is no fool-proof answer to this. You will have to tradeoff based on considerations, few of which I have highlighted above.

**Comment by samjameswaller** [comment]
Thanks Varun,

Much appreciated
